examples/vision/md/randaugment.md

```diff
@@ -0,0 +1,421 @@
+# RandAugment for Image Classification for Improved Robustness
+
+**Author:** [Sayak Paul](https://twitter.com/RisingSayak)<br>
+**Date created:** 2021/03/13<br>
+**Last modified:** 2021/03/13<br>
+**Description:** RandAugment for training an image classification model with improved robustness.
+
+
+<img class="k-inline-icon" src="https://colab.research.google.com/img/colab_favicon.ico"/> [**View in Colab**](https://colab.research.google.com/github/keras-team/keras-io/blob/master/examples/vision/ipynb/randaugment.ipynb)  <span class="k-dot">•</span><img class="k-inline-icon" src="https://github.com/favicon.ico"/> [**GitHub source**](https://github.com/keras-team/keras-io/blob/master/examples/vision/randaugment.py)
+
+
+
+Data augmentation is a very useful technique that helps to improve the translational
+invariance of convolutional neural networks (CNN). RandAugment is a stochastic data
+augmentation routine for vision data and was proposed in
+[RandAugment: Practical automated data augmentation with a reduced search space](https://arxiv.org/abs/1909.13719).
+It is composed of strong augmentation
+transforms like color jitters, Gaussian blurs, saturations, etc. along with more
+traditional augmentation transforms such as random crops.
```

Comment:
Great paper + example.

Suggestion: refactor to be more specific from the get-go

- "Data augmentation" -> "RandAugment... data augmentation"
- Add `(Cubik et al., 2019)` to add the first author + year of publication

```suggestion
RandAugment is a stochastic data augmentation routine for vision data that helps improve the translational invariance of convolutional neural networks (CNN). It was proposed in
[RandAugment: Practical automated data augmentation with a reduced search space](https://arxiv.org/abs/1909.13719) (Cubik et al., 2019).
It is composed of strong augmentation transforms, including color jitters, Gaussian blurs, saturations, along with more traditional augmentation transforms, such as random crops.
```